content/docs/matrix-concepts/end-to-end-encryption/_index.md

```diff
@@ -390,8 +390,8 @@ and store the details of the outbound session for future use.
 The client should then call `olm_outbound_group_session_id` to get the unique ID
 of the new session, and `olm_outbound_group_session_key` to retrieve the
 current ratchet key and index. It should store these details as an inbound
-session, just as it would when [receiving them via an m.room_key event]
-(#handling-an-mroom_key-event).
+session, just as it would when 
+[receiving them via an m.room_key event](#handling-an-mroom_key-event).
```

Comment:
```suggestion
[receiving them via an m.room_key event](#handling-an-m-room-key-event).
```

Apparently zola slightly differently does id's :) This seems to be what the real id is now.